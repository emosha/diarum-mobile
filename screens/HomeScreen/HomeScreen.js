import React, { Fragment } from 'react';
import { Text } from 'react-native';

export default class HomeScreen extends React.Component {
  render() {
    return (
      <Fragment>
        <Text>Home</Text>
      </Fragment>
    );
  }
}
