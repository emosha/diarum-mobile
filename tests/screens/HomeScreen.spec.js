import React from 'react';

import HomeScreen from '../../screens/HomeScreen/HomeScreen';

describe('HomeScreen', () => {
  let Component;

  beforeAll(() => {
    Component = mount(<HomeScreen />);
  });

  it('renders correctly', () => {
    expect(Component).toMatchSnapshot();
  });
});
