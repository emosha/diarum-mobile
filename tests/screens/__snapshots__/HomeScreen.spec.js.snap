// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen renders correctly 1`] = `
<HomeScreen>
  <Component>
    <Text>
      Home
    </Text>
  </Component>
</HomeScreen>
`;
